static/js/components/dashboard/CoursePrice.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+import React from 'react';
+import _ from 'lodash';
+
+import type { Course } from '../../flow/programTypes';
+import {
+  STATUS_ENROLLED_NOT_VERIFIED,
+  STATUS_OFFERED_NOT_ENROLLED,
+} from '../../constants';
+import { formatPrice } from '../../util/util';
+
+export default class CoursePrice extends React.Component {
+  props: {
+    course: Course,
+  };
+
+  render() {
+    const { course } = this.props;
+    let firstRun = {};
+    let price = null;
+    let enrollmentStatus = null;
+
+    if (course.runs.length > 0) {
+      firstRun = course.runs[0];
+      if (course.status === STATUS_OFFERED_NOT_ENROLLED || course.status === STATUS_ENROLLED_NOT_VERIFIED) {
+        if (!_.isNil(firstRun.price)) {
+          price = <span className="course-price-display">{formatPrice(firstRun.price)}</span>;
+        }
+        enrollmentStatus = <span className="course-enrollment-description">Enrollment open</span>;
```

Comment:
you mean remove 'open enrollment'  text only?

On Sep 2, 2016 7:19 PM, "George Schneeloch" <notifications@github.com>
wrote:

> In static/js/components/dashboard/CoursePrice.js
> <https://github.com/mitodl/micromasters/pull/960#discussion_r77353535>:
>
> > +    course: Course,
> > +  };
> > +
> > +  render() {
> > +    const { course } = this.props;
> > +    let firstRun = {};
> > +    let price = null;
> > +    let enrollmentStatus = null;
> > +
> > +    if (course.runs.length > 0) {
> > +      firstRun = course.runs[0];
> > +      if (course.status === STATUS_OFFERED_NOT_ENROLLED || course.status === STATUS_ENROLLED_NOT_VERIFIED) {
> > +        if (!_.isNil(firstRun.price)) {
> > +          price = <span className="course-price-display">{formatPrice(firstRun.price)}</span>;
> > +        }
> > +        enrollmentStatus = <span className="course-enrollment-description">Enrollment open</span>;
>
> It looks like there's more to #925
> <https://github.com/mitodl/micromasters/issues/925> than just showing
> this text. Can you remove it altogether and it will be added as part of
> #925 <https://github.com/mitodl/micromasters/issues/925>?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mitodl/micromasters/pull/960/files/3f49be52ac38730a3d7abda0559524b8bc82e798#r77353535>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AJ8rEhK7TrFO2dwF0EYD_QYpXn4T28zeks5qmDBxgaJpZM4Jyqep>
> .
>